### ISSUE TITLE
new proxmox version update undid dark theme

### ISSUE THREAD

**Issue by wommy** [Weilbyte/PVEDiscordDark#142]
i like their new disabled VM transparency, very clear which are running / not running

**Comment by Weilbyte** [comment]
Hi, I have not upgraded my servers to the new version yet, could you provide a screenshot of what you mean? There will be some time before I can be able to look at what exactly is changed and what I need to add.

**Comment by supersnellehenk** [comment]
That person probably means this: https://imgur.com/a/SHV81jl

**Comment by Weilbyte** [comment]
I see, thanks. There are a bunch of things I would also like to fix. Will attempt to catch up to the latest release soon (probably either this weekend or the next).

**Comment by djraw** [comment]
Today I tried some things and found what needs to be done to get grey icons for stopped VMs, lxc, unavailable storage and changes to template icons.
@Weilbyte : since you wrote that you want fix other things, I didn't add those changes to my fork and didn't add a pull request. Also I did this using Firefox dev mode and the resulting css is not compressed.. anyway, here you go (file PVEDiscordDark.css, resulting in dd_style.css):

--------------------
NEW
-------------------------
.fa-fw.x-grid-icon-custom.fa.fa-cube.stopped {
 color:#2f3d72
}

.x-tree-icon.x-tree-icon-custom.x-tree-icon-leaf.fa.fa-desktop.stopped {
 color:#555 !important
}

.fa-fw.x-grid-icon-custom.fa.fa-desktop.stopped {
 color:#555
}

.fa-fw.x-grid-icon-custom.fa.fa-database.available {
 color:#526DD1
}

--------------------
CHANGE
--------------------

.x-tree-icon.x-tree-icon-custom.x-tree-icon-leaf.fa.fa-cube {
 color:#555 !important
}

.x-tree-icon.x-tree-icon-custom.x-tree-icon-leaf.fa.fa-desktop {
 color:#555 !important
}

.x-tree-icon.x-tree-icon-custom.x-tree-icon-leaf.fa.fa-file-o:after {
 color:#555;
 background:transparent;
 text-shadow:none
}

.fa-fw.x-grid-icon-custom.fa.fa-file-o:after {
 color:#555;
 background:transparent;
 text-shadow:none
}

.fa-fw.x-grid-icon-custom.fa.fa-database {
 color:#555
}

Result:
![2022-09-28 00-45-30_Window](https://user-images.githubusercontent.com/10433914/192651469-bc768ec5-e8d1-4910-9769-f2cb0a2ab44b.png)

**Comment by Weilbyte** [comment]
Thanks, will get these changes in soon.

**Comment by johntdavis84** [comment]
Thanks to you both for working on this.

Once I realized I was back in light mode, I figured the last update had killed the theme and decided to remove it to avoid any issues. 

The uninstall command says there's no theme installed.
```
#] bash PVEDiscordDark.sh uninstall
Theme not installed
```

Will I need to do anything to clean out the old theme files before re-running the script when the theme is updated?

Thanks!

**Comment by Weilbyte** [comment]
@djraw only a browser cache clear should be necessary.

**Comment by djraw** [comment]
> @djraw only a browser cache clear should be necessary.

Yes, I forgot my script also updates certificates which makes the restart of pve-proxy necessary.

But anyway I'm right about the updates to pve-manager, after that a reinstall of the theme is needed. And that might be added to the readme.

**Comment by Weilbyte** [comment]
@wommy @johntdavis84 @djraw reinstalling the theme should fetch these latest changes. Let me know if it works.

**Comment by wommy** [comment]
looks great! so glad i complained :rofl: :rofl: :rofl: 

didnt know it was all sass,
next time i can totally contrib

**Comment by Weilbyte** [comment]
I think it looks ok as well but my only concern is how it will look on older versions, which I will attempt to test soon.

And yeah, it's all sass; it's still spaghetti but at least more manageable than if it was pure css 😓

**Comment by wommy** [comment]
since [css custom properties](https://piccalil.li/tutorial/getting-started-with-css-custom-properties/), its been much more managable

i could help with a migration
i noticed a bunch of little small things after peeking for a minute

**Comment by djraw** [comment]
> @wommy @johntdavis84 @djraw reinstalling the theme should fetch these latest changes. Let me know if it works.

Works perfect for me, thanks @Weilbyte :)

**Comment by Weilbyte** [comment]
@djraw  I will look into it soon. Added a comment on your PR.
